src/xdg.c

```diff
@@ -118,6 +156,11 @@ handle_destroy(struct wl_listener *listener, void *data)
 	wl_list_remove(&xdg_toplevel_view->set_app_id.link);
 	wl_list_remove(&xdg_toplevel_view->new_popup.link);
 
+	if (view->pending_configure_timeout) {
+		wl_event_source_remove(view->pending_configure_timeout);
+		view->pending_configure_timeout = NULL;
```

Comment:
Should we reset the serial to `0` as well here?

Reply:
It shouldn't matter since the view is being destroyed (even setting `view->pending_configure_timeout = NULL` should be technically unnecessary, and is only there as a safeguard).

Reply:
Right.. its the destroy handler. Somehow missed that earlier.